src/clientsettingsdlg.cpp

```diff
@@ -35,6 +35,16 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
 {
     setupUi ( this );
 
+#if defined ( Q_OS_IOS )
+    // IOS needs menu to close
+    QMenuBar* pMenu     = new QMenuBar ( this );
```

Comment:
There's maybe a bit too much space here:
```suggestion
    QMenuBar* pMenu = new QMenuBar ( this );
```